### PR TITLE
Add basic github connector

### PR DESCRIPTION
**Pull request by DenisPopov15**
Please check this basic github logic

### CODE REVIEW

File: connectors/github/src/api/callback.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+'use strict'
+
+const GithubService = require('../services/githubService')
+
+const callback = async(req, res) => {
+  const { code, did } = req.query
+
+  // const githubService = await GithubService.initialize(code)
```
Comment (dmfilipenko) [inline connectors/github/src/api/callback.js:8]: Should we remove comments?

File: connectors/github/src/App.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  createApi() {
+    this._logger.info('> createApi')
+    this.http.use('/api', routes)
+  }
+
+  createServerMiddleware() {
+    this._logger.info('> createServerMiddleware')
+
+    this.http.use(middleware.helmet())
+    // this.http.use(middleware.auth(this))
```
Comment (dmfilipenko) [inline connectors/github/src/App.js:68]: Remove comments

File: connectors/github/test/index.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+describe('API', () => {
+  require('./api/index.spec')
+})
+
+describe('Helpers', () => {
+  require('./helpers/createAuth.spec')
+})
+
+// describe('Middlwares:', () => {
```
Comment (dmfilipenko) [inline connectors/github/test/index.js]: Please remove comments

**Review by dmfilipenko — CHANGES_REQUESTED** [review]

File: connectors/github/src/api/callback.js (hunk 1 above)
Comment (DenisPopov15) [reply connectors/github/src/api/callback.js:8]: this is the main logic, so its should be there

File: connectors/github/test/index.js (hunk 3 above)
Comment (DenisPopov15) [reply connectors/github/test/index.js]: make sense, fixing

**Review by dmfilipenko — APPROVED** [review]
LGTM

**Pull request merged**